test/reactionsystem_01.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
save the test files in `test/data/` so that they don't mix up with code.

(That reminds me I should properly .gitignore the downloaded files.)

Reply:
fixed with 69fa2ff